### ISSUE TITLE
Implementation of Linear regression in Python

### ISSUE THREAD

**Issue by shrishiva16** [Kushal997-das/Project-Guidance#264]
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.)
Linear regression model built from scratch for easy understanding of concepts

## Motivation

(Please outline the motivation for the proposal.)
It is vital to understand the basics of machine learning basics

## Pitch

(Please explain why this feature should be implemented and how it would be used.)
Linear regression - definition
Gradient descent 
Learning rate
All will be explained along with the code
